zetaclient/chains/sui/client/client.go

```diff
@@ -10,7 +10,10 @@ import (
 
 	"github.com/block-vision/sui-go-sdk/models"
 	"github.com/block-vision/sui-go-sdk/sui"
+	patsui "github.com/pattonkan/sui-go/sui"
```

Comment:
We use this library for PTB, can we use suiptb as an alias instead

Reply:
https://github.com/zeta-chain/node/pull/3793/commits/003d155c0d4d41ff7a80e43e697d2256ebc89bb9